### ISSUE TITLE
Error Uploading CSV

### ISSUE THREAD

**Issue by milafrerichs** [davbre/mira#9]
Hello,

thanks for mira. Looks really promising. 
Was trying it out but ran into a problem. 
After uploading a `datapackage.json` which is valid according to the validator.
But when I upload a csv file the worker fails with:

```
Job ProcessCsvUpload (id=8) FAILED (0 prior attempts) with ActiveRecord::UnknownAttributeError: unknown attribute 'DATE' for Xy2_2.
```

Here is the excerpt from the datapackage.json
```json
"schema": {
        "fields": [
          {
            "name": "DATE",
            "type": "datetime"
          }
....
```
the api from mira for `datapackage/fields` returns
```json
[
{
"name": "DATE",
"order": 1,
"big_integer": null,
"add_index": true,
"format": null,
"type": "datetime"
},
...
```

Any help would be appreciated.
Thanks.

**Comment by davbre** [comment]
@milafrerichs - could you provide me with a link to your datapackage.json
file and the corresponding csv file(s), and I'll take a look?

**Comment by milafrerichs** [comment]
sure. here you go:
https://github.com/CivicVision/dwh_airmonitoring

**Comment by davbre** [comment]
Thanks - I was able to reproduce the error and I've updated the repository to fix this issue.

I am now able to upload your csv file using the slow upload. The quick upload still fails and this is due to the presence of "ND" values in the "result" column which is of type "number".

On my laptop the slow upload method takes 12 minutes. However, if I tweak your csv file by removing the "ND" values in the "result" column, then the quick upload works and takes less than 10 seconds, so that might be worth considering.

Let me know if you get it working.
